src/gui/tray/talkreply.cpp

```diff
@@ -0,0 +1,50 @@
+#include "talkreply.h"
+#include "accountstate.h"
+
+#include <QJsonDocument>
+#include <QJsonObject>
+#include <QJsonArray>
+
+namespace OCC {
+
+Q_LOGGING_CATEGORY(lcTalkReply, "nextcloud.gui.talkreply", QtInfoMsg)
+
+TalkReply::TalkReply(AccountState *accountState, QObject *parent)
+    : QObject(parent)
+    , _accountState(accountState)
+{
+}
+
+void TalkReply::sendReplyMessage(const QString &conversationToken, const QString &message, const QString &replyTo)
+{
+    Q_ASSERT(_accountState && _accountState->account());
+    
+    if (!_accountState || !_accountState->account()) {
+        return;
+    }
```

Comment:
why not checking the validity of `_accountState` in the constructor ?
would be easier to figure out in case of a problem

Reply:
just felt that this was relevant when trying to send a message but in the end the way I am doing it, it doesn't matter.